types/arraystat/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for arraystat 1.7
+// Project: https://www.npmjs.com/package/arraystat
+// Definitions by: Julian Wiche <https://github.com/dajuly20>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+
+interface ArrayStatType {
+    sum: number;
+    nb: number;
+    avg: number;
+    stddev: number;
+    min: number;
+    q1: number;
+    median: number;
+    q3: number;
+    max: number;
+    range: number;
+    histogram: Histogram[];
+}
+
+interface Histogram {
+    min: number;
+    max: number;
+    nb: number;
+}
+
+/**
+ * Executes a task as soon as possible.
+ * @param task Function or any object that implements `call()`.
+ */
+declare function arraystat(data: number[]): ArrayStatType;
```

Comment:
this module seems to ship only ES6 export for a single function, check here:
https://unpkg.com/browse/arraystat@1.7.75/index.js
The template used here was for UMD, please use ES6 module template:
https://www.typescriptlang.org/docs/handbook/modules.html#zipcodevalidatorts-1

Reply:
Yeah, this should just be `export default function arraystat` and no `export =`.